src/Sulu/Bundle/MediaBundle/Tests/Application/config/config_secured.yml

```diff
@@ -0,0 +1,24 @@
+security:
+    enable_authenticator_manager: true
```

Comment:
@niklasnatter I had to add this line else the test did not fail in my case.

Reply:
thats interesting, i think this should always be set to true on Symfony 6: https://github.com/symfony/symfony/blob/5.4/UPGRADE-6.0.md 
but shouldnt hurt to set it explicitly 🙂

Reply:
The 2.4 branch did run on `4.4` and `5.4` the default there is `enable_authenticator_manager: false`.